readme.md

```diff
@@ -260,8 +323,7 @@ combineReducers({
 });
 ```
 
-you can pass `defaultFlags` as the `preloadedState` directly into the `flopflipReducer` and do not need to
-keep track of it in your applications's `initialState` as in
+This way you can pass `defaultFlags` as the `preloadedState` directly into the `flopflipReducer`. This means you do not need to keep track of it in your applications's `initialState` as in the following anti-pattern example:
```

Comment:
Because from what I understand, initialState in redux serves as an entrypoint for server-side state hydration. It's important when you do server side rendering. In the context of a SPA without state hydration I feel like it's a misuse of the method.